contributors.md

```diff
@@ -5,250 +5,432 @@ title: Contributors
 # `website` are optional. If you provide an `email` and a `website`, the
 # name will link to the website.
 contributors:
-  - name: John Abbott
+
+
+    ######################
+    # Project leads
+    ######################
+
+  - name: Wolfram Decker
     affiliation: University of Kaiserslautern-Landau
-    paid_by_dfg: true
+    website: https://math.rptu.de/en/wgs/agag/people/head/decker
+    email: wolfram.decker@rptu.de
+    github: wdecker
+    is_active_PI: true
+
+  - name: Claus Fieker
+    affiliation: University of Kaiserslautern-Landau
+    website: https://math.rptu.de/en/wgs/agag/people/head/fieker
+    email: claus.fieker@rptu.de
+    github: fieker
+    is_active_PI: true
+
+  - name: Max Horn
+    affiliation: University of Kaiserslautern-Landau
+    website: https://www.quendi.de/math
+    email: mhorn@rptu.de
+    github: fingolfin
+    is_active_PI: true
+
+  - name: Michael Joswig
+    affiliation: TU Berlin
+    website: https://page.math.tu-berlin.de/~joswig/
+    email: joswig@math.tu-berlin.de
+    github: micjoswig
+    is_active_PI: true
+
+
 
-  - name: Mohamed Barakat
-    affiliation: University of Siegen
-    website: https://mohamed-barakat.github.io/
 
-  - name: Reimer Behrends  # parallelization, GAP garbage collector interface
-    affiliation: TU Kaiserslautern
-    retired: true
-    #github: rbehrends
 
-  - name: Alex Best
-    affiliation: Boston University
+    ######################
+    # Active contributors
+    ######################
+
+  - name: John Abbott
+    affiliation: University of Kaiserslautern-Landau
+    website: https://math.rptu.de/ags/agag/personen/mitglieder
+    email: john.abbott@rptu.de
+    github: JohnAAbbott
+    paid_by_dfg: true
+
+  - name: Alex J. Best
+    affiliation: King’s College London
+    website: https://alexjbest.github.io/
+    email: alex.j.best@gmail.com
     github: alexjbest
 
   - name: Martin Bies
     affiliation: University of Kaiserslautern-Landau
     website: https://martinbies.github.io
+    email: martin.bies@rptu.de
```

Comment:
Actually seeing this now: I am not sure we should add email addresses for everyone, this might turn into a privacy / data protection headache, at least if those email addresses are not otherwise public (maybe they all are in the git commit logs, though?)

It should be fine for the PIs. Dunno

Reply:
All emails are public in at least one way. Say by visiting the linked website and then looking up the email there. Once could (in my case) also go through my github profile, which will again link to my website.

Still, if you feel I should remove emails, then this is of course an easy fix.

For all it matters - I have also added emails for all OSCAR book contributors. So this issue repeats for the OSCAR book website/contributors.